clinicadl_zoo/transforms/factory.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Union
+
+# pylint: disable=unused-import
+from clinicadl.transforms.config.base import TransformConfig
+
+# factory of custom transforms
+from .config.nan_removal import NanRemovalConfig
+from .config.template import TemplateConfig
+from .enum import ZooTransform
+
+
+def get_transform_config(
+    name: Union[str, ZooTransform], **kwargs: Any
+) -> TransformConfig:
+    """
+    Factory function to get a transform configuration object from its name
+    and parameters.
+
+    Parameters
+    ----------
+    name : Union[str, ZooTransform]
+        the name of the transform. Check our documentation to know
+        supported transforms.
+    **kwargs : Any
+        any parameter of the transform. Check our documentation on transforms to
+        know these parameters.
+
+    Returns
+    -------
+    TransformConfig
+        the config object. Default values will be returned for the parameters
+        not passed by the user.
+    """
+    transform = ZooTransform(name)
+    config_name = f"{transform.value}Config"
+    try:
+        config = globals()[config_name]
+    except KeyError:
+        raise ValueError(
+            f"Can't find the transform {name}. "
+            f"Please check if you have imported the transform in the config factory."
```

Comment:
```suggestion
            f"Can't find the config class associated to the transform '{name}'. "
            "Please check that you have created this config class."
```

Just a suggestion. WDYT?